webpack.config.js

```diff
@@ -18,16 +18,13 @@ module.exports = {
     // Entry points locations.
     entry: {
         [`${paths.package.name}-css`]: `${__dirname}/${paths.scssEntry}`,
-        [`${paths.package.name}-js`]: `${__dirname}/${paths.jsEntry}`,
-
         'admin_overrides': `${__dirname}/${paths.scssSrcDir}/admin/admin_overrides.scss`,
     },
 
     // (Output) bundle locations.
     output: {
         path: __dirname + '/' + paths.jsDir,
-        filename: '[name].js', // file
-        chunkFilename: '[name].bundle.js',
+        filename: '[name].js',   // file
```

Comment:
I see that `npm run build && src/manage.py collectstatic --noinput` still outputs some files in `static/bundles/` like `openklant-css.css`, `openklant-css.js` and ``openklant-js.js`. Could you check if these are still used in any templates? Otherwise we might be able to remove those from this webpack config as well